turbo.json

```diff
@@ -55,12 +55,19 @@
         ".output/**"
       ]
     },
-    "generate:component-statuses": {
-      "cache": false,
+    "//#generate:component-statuses": {
```

Comment:
Think this is fine as this task is only defined in this root config.

```suggestion
    "generate:component-statuses": {
```

Reply:
Tested this and it seems to be necessary